scale-ui/app/modules/jobs/controllers/jobsController.js

```diff
@@ -79,7 +79,13 @@
             {
                 field: 'status',
                 width: 150,
-                cellTemplate: '<div class="ui-grid-cell-contents"><div class="pull-right"><button ng-show="((!grid.appScope.vm.readonly) && (row.entity.status === \'FAILED\' || row.entity.status === \'CANCELED\'))" ng-click="grid.appScope.vm.requeueJobs({ job_ids: [row.entity.id] })" class="btn btn-xs btn-default" title="Requeue Job"><i class="fa fa-repeat"></i></button> <button ng-show="!grid.appScope.vm.readonly && row.entity.status !== \'COMPLETED\' && row.entity.status !== \'CANCELED\'" ng-click="grid.appScope.vm.cancelJob(row.entity)" class="btn btn-xs btn-default" title="Cancel Job"><i class="fa fa-ban"></i></button></div> {{ row.entity.status }}</div>',
+                cellTemplate: '<div class="ui-grid-cell-contents">' +
+                    '<div class="pull-right">' +
+                        '<button ng-show="((!grid.appScope.vm.readonly && !row.entity.is_superseded) && (row.entity.status === \'FAILED\' || row.entity.status === \'CANCELED\'))" ng-click="grid.appScope.vm.requeueJobs({ job_ids: [row.entity.id] })" class="btn btn-xs btn-default" title="Requeue Job">' +
+                            '<i class="fa fa-repeat"></i></button> ' +
+                        '<button ng-show="!grid.appScope.vm.readonly && !row.entity.is_superseded && row.entity.status !== \'COMPLETED\' && row.entity.status !== \'CANCELED\'" ng-click="grid.appScope.vm.cancelJob(row.entity)" class="btn btn-xs btn-default" title="Cancel Job">' +
+                            '<i class="fa fa-ban"></i></button>' +
+                    '</div> {{ row.entity.status }} <span ng-if="row.entity.is_superseded" class="label label-info">SUPERSEDED</span> </div>',
```

Comment:
I think that's better. Since it is somewhat related to "status" information and I don't believe a superseded job can have any actions it might be better to just place it where the action buttons would have been. If we leave it in it's current position it takes away more space from the title. Either way, make sure it doesn't conflict with longer titles and that it has a tool-tip.